scalafix/rules/src/main/scala/fix/Scalacollectioncompat_newcollections.scala

```diff
@@ -156,6 +163,28 @@ case class Scalacollectioncompat_newcollections(index: SemanticdbIndex)
         ctx.replaceTree(t, "lazyAppendedAll")
     }.asPatch
 
+  def replaceSetMapPlus2(ctx: RuleCtx): Patch = {
+    def rewritePlus(ap: Term.ApplyInfix, lhs: Term, op: Term.Name, rhs1: Term, rhs2: Term): Patch = {
+      ctx.replaceTree(
+        ap,
+        Term.ApplyInfix(
+          Term.ApplyInfix(lhs, op, Nil, List(rhs1)),
+          op,
+          Nil,
+          List(rhs2)
+        ).toString
```

Comment:
What does `toString` do? That looks suspect :smile:

Reply:
It's fine, it will drop any comments. It's the scope of https://github.com/scalacenter/scala-syntax

Reply:
I was wondering whether `toString` was the recommended way to produce scala source code from a tree or if there was a method with a more meaningful name.